__tests__/Auth0Client/getTokenSilently.test.ts

```diff
@@ -189,7 +189,9 @@ describe('Auth0Client', () => {
     });
 
     it('calls the token endpoint with the correct params', async () => {
-      const auth0 = setup();
+      const auth0 = setup({
+        useFormData: false
+      });
```

Comment:
This is a general consideration that counts for a lot of the changes in this PR.

From a technical perspective, tests that are updated in the same way as here, are still testing the same thing as before.

However, they are not testing the same user interaction anymore, for example, this test used to test the public API:

```
new Auth0Client()
```
While not, it actualy tests 
```
new Auth0Client({useFormData: false})
```
This counts for any of our tests that build on top of that client, they are all now assuming the client is created differently (even tho still using the same internal details).

As `useFormData` is a feature added later, I do not think we cover every feature in our tests with using formData. (if we would, just flipping the switch in all tests would be all it takes)

Even tho I believe the change here is great, flipping that switch in the tests would mean the majority of our tests are now for our users that use `new Auth0Client({useFormData: false})`. So I have a feeling we are missing out on some important test cases.

Would it make sense to look at any test that now has `useFormData: false`, to see if the counter-case is tested, and if not add a copy of the test that tests using form-data?

Reply:
Yeah that makes sense, I'll work through where I've added `useFormData: false` and ensure that a test case exists that is also testing with `useFormData: true` (i.e. no explicit `useFormData` option)

Reply:
Added tests that cover the new `useFormData` default of true